dotcom-rendering/src/web/components/DecideLines.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticlePillar, ArticleSpecial } from '@guardian/libs';
+import {
+	DottedLines,
+	StraightLines,
+} from '@guardian/source-react-components-development-kitchen';
+import { GuardianLabsLines } from './GuardianLabsLines';
+
+type Props = {
+	format: ArticleFormat;
+	color?: string;
+	useLabsLines?: boolean;
+};
+
+export const DecideLines = ({ format, color, useLabsLines }: Props) => {
```

Comment:
This isn't necessary where it's used [in the ImmersiveLayout](https://github.com/guardian/dotcom-rendering/pull/5233/files#diff-10db6a9a97924d5e0c6e781cacae999645361d4aba55cace3f4bae9ce3d57666R328-R337) and [the InteractiveImmersiveLayout](https://github.com/guardian/dotcom-rendering/pull/5233/files#diff-10db6a9a97924d5e0c6e781cacae999645361d4aba55cace3f4bae9ce3d57666R328-R337) as there is a `format.theme === ArticleSpecial.Labs` guard in those files.

It's only [in the StandardLayout](https://github.com/guardian/dotcom-rendering/pull/5233/files#diff-c9298775646478b23d69b23a9e0b4c5e3b1a5daf67529f11a2b62317b516f662R586-R589) that we need this, creating two ways to get `GuardianLabsLines`, which is potentially confusing.

Could we potentially move all the conditions into `DecideLines`, and allow it to decide to render no lines (`<></>`, meaning it would also needs surrounding HTML<sup>*</sup>), or add the `format.theme === ArticleSpecial.Labs` guard to the `StandardLayout` and remove this override prop?


<sup>*</sup>surrounding HTML
```jsx
<div css={maxWidth}>
  <div css={stretchLines}>
  </div>
</div>

```

Reply:
I shall re-investigate this logic! I want to be careful not to over-abstract and turn it into an overly complex component!

Reply:
> creating two ways to get GuardianLabsLines, which is potentially confusing.

I'm a tad confused what you mean by this! Are you able to elaborate?

Reply:
After going over this with @jamesgorrie we decided that the cleanest implementation kept the `<GuardianLabsLines />` component in the layout files